packages/inventory/src/services/inventory-item.ts

```diff
@@ -120,33 +120,35 @@ export default class InventoryItemService {
    */
   @InjectEntityManager()
   async create(
-    data: CreateInventoryItemInput,
+    data: CreateInventoryItemInput[],
```

Comment:
todo: As we discussed, we can allow an item or a collection wdyt?

Reply:
This is not publicly exposed, this is the service for only inventory-items and is only used through the api hence why it's just bulk.